templates/data/generic_component_kyma_dashboard_data.yaml

```diff
@@ -10,7 +10,7 @@ templates:
                   args:
                     - "."
                     - "release-dev"
-                  run_if_changed: "^Dockerfile.*|^package.json|^environments/dev/|^extensions|^gulpfile.js|^Makefile"
+                  run_if_changed: "^Dockerfile.*|^package.json|^environments/dev/|^extensions|^gulpfile.js|^Makefile|^tests/"
```

Comment:
can we add smoke test files for stage and dev jobs?

Reply:
for dev we are running whole integration tests, so smoke tests are unnecessary. for stage adjusted